dnf5/commands/repoquery/repoquery.cpp

```diff
@@ -306,6 +326,46 @@ void RepoqueryCommand::set_argument_parser() {
     disable_modular_filtering = std::make_unique<libdnf5::cli::session::BoolOption>(
         *this, "disable-modular-filtering", '\0', "Include packages of inactive module streams.", false);
 
+    // Allowed values for --providers-of options (these package attributes return ReldepLists)
+    std::vector<std::string> pkg_attrs_options{
+        "conflicts",
+        "depends",
+        "enhances",
+        "obsoletes",
+        "provides",
+        "recommends",
+        "requires",
+        "requires_pre",
+        "suggests",
+        "supplements",
+        "",  // empty when option is not used
```

Comment:
I see some extra handling of this empty option on two places below in this file. Could it be easier to instantiate the `providers_of_option` and `pkg_attr_option`, then remove this last empty item from the vector and after that use it for `allowed_values` and the iteration on L418?

Reply:
I think its not possible because the `pkg_attr_option` and the iteration on L418 also accept 3 additional possibilities `{"file", "sourcerpm", "location"}` - these are prepended to the vector of `pkg_attrs_options` before the `pkg_attr_option` is created but these cannot be in the `allowed_values`.

Reply:
Ok, let's leave it like that. Maybe I'd just drop the comments with `// Remove last empty space` and `// change last ", " to '.'` and put there sth like `// Drop the empty option from the description of supported values`?

Reply:
That sound good, updated. :slightly_smiling_face: